devimint/src/federation.rs

```diff
@@ -170,18 +170,28 @@ impl Federation {
 
         run_dkg(admin_clients, params).await?;
 
-        let out_dir = &vars[&0].FM_DATA_DIR;
         let cfg_dir = &process_mgr.globals.FM_CLIENT_DIR;
-        let out_dir = utf8(out_dir);
-        let cfg_dir = utf8(cfg_dir);
-        // move configs to config directory
-        tokio::fs::rename(
-            format!("{out_dir}/invite-code"),
-            format!("{cfg_dir}/invite-code"),
-        )
-        .await
-        .context("moving invite code file")?;
-        info!("moved client configs");
+        let cfg_dir: &str = utf8(cfg_dir);
+        let invite_code_filename_original = String::from("invite-code");
+        for (index, var) in &vars {
+            let out_dir: &PathBuf = &var.FM_DATA_DIR;
+            let out_dir = utf8(out_dir);
+            // move configs to config directory
+
+            let invite_code_filename_indexed = format!("invite-code-{}", index);
+            let invite_code_filename_new: String = if index == &0 {
```

Comment:
Is there a reason why we're special casing index 0? Seems to me the code here and below would be simpler if it just always expected the files to be named `invite-code-X`, even for the 0th peer

Reply:
I definitely agree it would be simpler. My thought process was that it would avoid any backwards compatibility issues that could arise, and there are a few other places the `invite-code` file was used so I didn't want to cause any issues:
- `fedimint-server/src/config/io.rs`
- `devimint/src/bin/faucet.rs`
- `devimint/src/cli.rs`

If these are irrelevant, then I can happily make the change to simplify the code.

Reply:
Make a copy in both places first, then as a follow-up we could get rid of places that use the old path.